2200107/Q2.py

```diff
@@ -0,0 +1,81 @@
+# 신용카드 만들기2
+accept_number=['3','4','5','6','9']
+T= int(input())
+for tc in range(T):
+    # input 카드 번호
+    numbers=list(input())
+    new=[]
+
+        # # 실패한 시도들. replace 메소드 사용법을 숙지 못해서... 사용하려다 실패.
+        # nli=[]
+        # for e in range(len(numbers)):
+        #     if numbers[e]=='-':
+        #        for i in numbers:
+        #         if i!='-':
+        #           nli.append(i)
+        # print(nli)
+        # print(numbers)
+        # print(nli) 
+        # print(numbers) 
+
+    # replace 메소드 사용법 몰라서 -를 제거한 list 생성. new list.
+    for rmv in numbers:
+        new.append(rmv)
+        if rmv=='-':
+            new.pop()
```

Comment:
```suggestion
    # replace 메소드 사용법 몰라서 -를 제거한 list 생성. new list.
    new = numbers.replace('-', '')
```
`replace` 함수는 위와 같이 써주시면 됩니다 😄